2024-04-25 16:43:00.6444 ERROR Catch exception:API error : User was not found.|| trace:85
2024-04-25 16:51:55.9826 ERROR Catch exception:API error : Category was not found.|| trace:59
2024-04-25 16:52:06.2117 ERROR Catch exception:API error : Duplicate name.|| trace:0
2024-04-25 16:52:25.5270 ERROR Catch exception:API error : An error occurred while saving the entity changes. See the inner exception for details.|| trace:0
2024-04-25 16:52:51.1240 ERROR Catch exception:API error : Category was not found.|| trace:94
2024-04-25 16:53:05.3620 ERROR Catch exception:API error : Product was not found.|| trace:70
2024-04-25 16:53:43.1167 ERROR Catch exception:API error : Product was not found.|| trace:141
2024-04-25 16:53:43.3495 ERROR Catch exception:API error : Product was not found.|| trace:141
2024-04-25 16:53:52.3724 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 16:54:56.0585 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:17:15.8008 ERROR Catch exception:API error : Duplicate name.|| trace:0
2024-04-25 17:21:58.9128 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:23:27.9718 ERROR Catch exception:API error : Category was not found.|| trace:148
2024-04-25 17:23:42.5485 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:28:37.2812 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:39:01.2490 ERROR Catch exception:API error : Category was not found.|| trace:148
2024-04-25 17:39:49.3821 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:44:55.1148 ERROR Catch exception:API error : Category was not found.|| trace:149
2024-04-25 17:45:36.5769 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:46:10.4554 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 17:48:04.7985 ERROR Catch exception:API error : The instance of entity type 'ProductEntity' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.|| trace:0
2024-04-25 18:06:59.4157 ERROR Catch exception:API error : Category was not found.|| trace:120
2024-04-25 18:07:03.1543 ERROR Catch exception:API error : Category was not found.|| trace:120
2024-04-25 18:07:48.8949 ERROR Catch exception:API error : Object reference not set to an instance of an object.|| trace:123
2024-04-25 18:08:06.6207 ERROR Catch exception:API error : Object reference not set to an instance of an object.|| trace:123
2024-04-25 18:08:12.3142 ERROR Catch exception:API error : Object reference not set to an instance of an object.|| trace:123
